app/views/idv/gpo/_address_on_file.html.erb

```diff
@@ -2,4 +2,4 @@
               idv_gpo_path,
               method: 'put',
               class: 'usa-button usa-button--big usa-button--wide',
-              form_class: 'inline-block margin-right-2' %>
```

Comment:
I removed this since it was preventing the button from being full width on mobile.

Reply:
> I removed the `inline-block` since it was preventing the button from being full width on mobile.

Nice catch. That seems right to me. I'd also go a step further and remove the margin, since we're not showing anything next to the button, and it's causing the mobile button to show as not-quite-full-width.

Screenshots:

Before|With `margin-right-2`|Without `margin-right-2`
---|---|---
![Screen Shot 2021-05-13 at 2 25 25 PM](https://user-images.githubusercontent.com/1779930/118169815-5fc56c80-b3f7-11eb-86f4-f09ec8523578.png)|![Screen Shot 2021-05-13 at 2 25 51 PM](https://user-images.githubusercontent.com/1779930/118169808-5cca7c00-b3f7-11eb-9b4b-60f17194a6c9.png)|![Screen Shot 2021-05-13 at 2 25 47 PM](https://user-images.githubusercontent.com/1779930/118169812-5dfba900-b3f7-11eb-86a0-0b9b6cd5d32d.png)

(cc @anniehirshman-gsa)

Reply:
I agree!

Reply:
Yup - done in 7f0e808